### DISCUSSION TITLE
How to get all the M365 platforms in a M365 group by powershell?

### DISCUSSION THREAD

**Discussion by khha** [pnp/powershell#4466] [Q&A]
A M365 group could have for example the platforms SharePoint, Exchange, Planner, Viva Engage, MS Teams. How do I know which platforms are in a specific M365 group by powershell? 

I know only how to do this to check if the M365 group contains a MS Team:

```
$site = Get-PnPSite -Includes GroupId
if($site.GroupId) {
    $group = Get-PnPMicrosoft365Group -Identity $site.GroupId
    $group.HasTeam
}
```

**Comment by Barbarur** [comment]
A MS365 group always has a mailbox (exchange) and a SharePoint site associated. MS teams is optional that's why it has a parameter.

Planner is a bit different because it's not a unique object associated to it (mailbox, site and team are unique for the group).
A group can be owner of several plans in planner, the same as can be owner of a several MS Froms, but there is not such association, the group is just the account owner.

I think each MS365 is at the same time a community on Viva Engage, but I'm not very sure of that.

**Reply by khha** [reply]
> A MS365 group always has a mailbox (exchange) and a SharePoint site associated. MS teams is optional that's why it has a parameter.
> 
> Planner is a bit different because it's not a unique object associated to it (mailbox, site and team are unique for the group). A group can be owner of several plans in planner, the same as can be owner of a several MS Froms, but there is not such association, the group is just the account owner.
> 
> I think each MS365 is at the same time a community on Viva Engage, but I'm not very sure of that.

Hi, thank you for your reply. An M365 group is not at the same time a community on Viva Engage. Otherwise you are right. An Viva Engage community is always an M365 group.
